bundle-size/Tabster.fixture.js

```diff
@@ -0,0 +1,45 @@
+import {
+    createTabster,
+    disposeTabster,
+    forceCleanup,
+    getTabsterAttribute,
+    setTabsterAttribute,
+    getGroupper,
+    getMover,
+    getCrossOrigin,
+    getDeloser,
+    getModalizer,
+    getObservedElement,
+    getOutline,
+    getCurrentTabster,
+    getInternal,
+    overrideBasics,
+    makeNoOp,
+    isNoOp,
+    Types,
+} from "tabster";
+
+console.log(
+    createTabster,
+    disposeTabster,
+    forceCleanup,
+    getTabsterAttribute,
+    setTabsterAttribute,
+    getGroupper,
+    getMover,
+    getCrossOrigin,
+    getDeloser,
+    getModalizer,
+    getObservedElement,
+    getOutline,
+    getCurrentTabster,
+    getInternal,
+    overrideBasics,
+    makeNoOp,
+    isNoOp,
+    Types
+);
+
+export default {
+    name: "tabster",
```

Comment:
Is this fixture useful? Is there sense to have something more granular?

Reply:
I guess having one fixture with all that's been exported from tabster is good to know the actual total bundle size. I'll add a more granular one also